config/routes.rb

```diff
@@ -32,6 +32,8 @@
 
   get "/terms", to: "about#terms", as: :terms_page
 
+  post "/developers/search", to: "developers#index"
```

Comment:
I think you can get rid of this with the `method: :get`, but definitely double check.